src/pulse3D/plate_recording.py

```diff
@@ -345,7 +345,7 @@ def __init__(
         self.path = path
         self.wells = []
         self._iter = 0
-        self.is_optical_recording = False
+        self.is_optical_recording = "xlsx" in self.path
```

Comment:
Is this value actually used? I think we only need to set this attr on the plate recording obj itself

Reply:
this is already being set inside plate recording